frontend/src/app/person/person.component.html

```diff
@@ -97,14 +135,14 @@ <h2>Situation fiscale</h2>
           </span>
         </ng-container>
       </p>
-    </ng-container>
   </div>
 </div>
-
-<a class="btn btn-outline-primary" [routerLink]="['/persons', person.id, 'edit']">
-  <span class="fa fa-edit"></span>
-  Éditer
-</a>
+<div class="mt-4">
```

Comment:
Too much space: the button ends up closer to the "Notes" title below than to the person information, especially given the separator.

Reply:
:+1: